lib/iris/tests/integration/test_grib_load.py

```diff
@@ -104,7 +104,15 @@ def test_polar_stereo_grib2(self):
         cube = iris.load_cube(tests.get_data_path(
             ("GRIB", "polar_stereo",
              "CMC_glb_TMP_ISBL_1015_ps30km_2013052000_P006.grib2")))
-        self.assertCML(cube, ("grib_load", "polar_stereo_grib2.cml"))
```

Comment:
@marqh Do we need to remove the cml file `iris/tests/results/grib_load/polar_stereo_grib2.cml` ?

What about checking the integrity of the other cube metadata? i.e. there's a `forecast_period`, `forecast_reference_time`, `pressure` and `time` coordinates?

We could could be more rigorous in checking the CRS metadata is equivalent to `<stereographic central_lat="90.0" central_lon="249.0" ellipsoid="GeogCS(6371229.0)" false_easting="0.0" false_northing="0.0" true_scale_lat="60.0"/>`